cookbook/docs/Makefile

```diff
@@ -4,7 +4,7 @@ rsts:
 
 .PHONY: html
 html:
-	sphinx-build -n -b html . _build
+	sphinx-build -n -b html . _build && sphinx-build -n -b doctest . _build
```

Comment:
does this fail if doctest fails?

Reply:
it does (tested locally). This tests code under the ``.. testcode::` and `.. testoutput::` directives